Aditional translation es_ES (#3780)
cosmos/cosmos-sdk: docs/gaia/launch/blog-1-sp.md

> # Cosmos Hub para iniciar Mainnet

Author: solved

> ## Dependencias previas al lanzamiento y cómo reclamar sus Atoms con seguridad

Author: On this [point](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/gaia/launch/blog-1-en.md#pre-launch-dependencies--how-to-safely-claim-your-atoms) I see this subtitle

> En el verano de 2016, se publicó el [Cosmos whitepaper](https://cosmos.network/resources/whitepaper). En la 

Author: ok,but in the [doc](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/gaia/launch/blog-1-en.md#pre-launch-dependencies--how-to-safely-claim-your-atoms) of Cosmos appears summer, and spring.
Do I leave it as is or change it to Q3?

> primavera de 2017, se completó la [recaudación de fondos de Cosmos](https://fundraiser.cosmos.network/). En los primeros 

Author: solved

meses de 2019, el software está [completo](https://github.com/cosmos/cosmos-sdk/releases). El lanzamiento del Hub del Cosmos se acerca. 
¿Qué significa esto para los poseedores de átomos?

Si eres poseedor de Atoms, podrás delegar Atoms en los validadores de la red principal y votar sobre las propuestas de 

> gobernanza. De hecho, el éxito futuro de la red depende de que lo haga de forma responsable. Sin embargo, aún no podrás 

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.
And with _gobernanza_  I think is more correct like _governación_
With the _"."_ instead of  _"!"_  in the documentation there's an exclamation

> transferir Atoms. Las transferencias se desactivarán a nivel de protocolo hasta que se ejecute un hard-fork para habilitarlas.

Author: In the documentation appears a stop _(punto y seguido)_ instead of a comma.

Los poseedores de Atoms deben seguir cuidadosamente las directrices para poder delegar Atoms de forma segura. Por favor, 

> lea primero toda la guía para familiarizarse antes de hacer nada: [Guía CLI](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/gaia/delegator-guide-cli.md)

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

El proceso descrito en la guía es actualmente la única forma verificada y segura de delegar Atoms en el lanzamiento. Esto se debe a que 

> la herramienta gaiacli utilizada en la guía es el único software de cartera que está siendo sometido a auditorías de seguridad 

Author: I don't think it's right.
Es la única **billetera que está ha sido sometida**, this isn't a good translate.
_billetera_ ?? is more correct _wallet_ or _cartera_, but no _billetera_

> por parte de terceros en este momento. Ningún otro proveedor de carteras ha iniciado aún las auditorías de seguridad.

Author: Is more correct wallet or _cartera_, but no _billetera_

de delegar!

The Cosmos Hub es un software altamente experimental. En estos primeros días, podemos esperar tener problemas, actualizaciones 
y errores. Las herramientas existentes requieren conocimientos técnicos avanzados e implican riesgos que están fuera del control 
de la Fundación Interchain y/o del equipo de Tendermint (ver también la sección de riesgos en [Interchain Cosmos Contribution Terms](https://github.com/cosmos/cosmos/blob/master/fundraiser/Interchain%20Cosmos%20Contribution%20Terms%20-%20FINAL.pdf)). 
Cualquier uso de este software de código abierto [con licencia Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) se hace bajo su propio riesgo y sobre una 
base "TAL CUAL" sin garantías ni condiciones de ningún tipo, y toda responsabilidad de la Fundación Interchain y/o del equipo 
de Tendermint por daños y perjuicios. que surjan en relación con el software. Por favor, haz ejercicio extremo de precaución!

Si buscas más información sobre la delegación y quieres hablar con la gente que está desarrollando Cosmos, únete a la reunión 
virtual del 14 de febrero, en la que se te explicarán paso a paso las instrucciones para delegar Atoms en el lanzamiento.

> Regístrese aquí: [gotowebinar.com/register/](https://register.gotowebinar.com/register/5028753165739687691)

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

## Hitos restantes para el lanzamiento

Para seguir el progreso del lanzamiento de mainnet, por favor, añádalo a sus favoritos: [cosmos.network/launch](https://cosmos.network/launch)

### 5 Cosmos-SDK Auditorías de seguridad ✔

> A principios de enero, el Cosmos-SDK se sometió a la primera de una serie de evaluaciones de seguridad de terceros programadas para el primer trimestre de 2019. Esta auditoría se llevó a cabo durante un período de dos semanas y media. Hasta la fecha, 

Author: The `Cosmos-SDK` is a software, so, in spanish is _el software_ no _la software_

dos empresas de auditoría de seguridad diferentes han evaluado varias partes del Cosmos-SDK y se está llevando a cabo una 
tercera auditoría.

> ### 4 Cosmos SDK Suspensión de Funciones

Author: I don't see `Extra feature` on [blog](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/gaia/launch/blog-1-en.md#4-cosmos-sdk-feature-freeze)

los ganadores. Estos serán anunciados después de la finalización del juego.

### 2 Transacciones para el Genesis recopiladas

La Fundación Interchain publicará una recomendación para la asignación de Atoms en el génesis. Esto incluirá asignaciones 
para los participantes de la recaudación de fondos de Cosmos, los primeros contribuyentes y los ganadores del Game of Stakes. 
Cualquier persona con una asignación recomendada tendrá la oportunidad de presentar un gentx, el cual es requerido para 
convertirse en validador de genesis. El resultado final de la asignación recomendada y la colección de gentxs es 
un [archivo de génesis](https://forum.cosmos.network/t/genesis-files-network-starts-vs-upgrades/1464) final.

### 1 Lanzamiento del Cosmos Hub en Mainnet

Una vez que un archivo de génesis es aprobado por la comunidad, y +⅔ del poder de voto se pone en línea, la red principal 
de Cosmos estará en vivo.

## Canales Oficiales de comunicacion de Cosmos

Estas son las cuentas oficiales que comunicarán los detalles del lanzamiento:

- [Cosmos Network](https://twitter.com/cosmos)
- [Cosmos GitHub](https://github.com/cosmos)
- [Cosmos Blog](https://blog.cosmos.network)

> Por favor, tenga en cuenta que el [foro](https://forum.cosmos.network/),[grupos de chat de RIOT](https://riot.im/app/#/group/+cosmos:matrix.org), y [grupo de Telegram](http://t.me/cosmosproject)

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

no deben ser tratados como noticias oficiales de Cosmos.

> Si tiene dudas o confusión sobre los próximos pasos a seguir y no está seguro sobre fuentes de información confiables, 

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

> no haga nada durante el período inicial y espere una actualización a través de los tres canales de comunicación mencionados 

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

anteriormente. No proporcione nunca sus 12 palabras a ningún administrador, sitio web o software no oficial.

**Nunca le pediremos su clave privada o sus palabras de recuperación.**

> ## Manténgase seguro (y protegido) para el lanzamiento de la Mainnet

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.
The change `Mainnet` to `red principal` is ok

infracciones de seguridad de las cuentas, y no sólo se producen a través del correo electrónico: en la actualidad, se 
producen intentos de phishing oportunistas y selectivos [en cualquier lugar en el que haya una bandeja de entrada](https://www.umass.edu/it/security/phishing-fraudulent-emails-text-messages-phone-calls). 
No importa si está usando Signal, Telegram, SMS, Twitter, o simplemente revisando sus DMs en foros o redes sociales, los 
atacantes tienen una [plétora de oportunidades](https://jia.sipa.columbia.edu/weaponization-social-media-spear-phishing-and-cyberattacks-democracy) para entrar en su vida digital en un esfuerzo por separarlo de la 
información valiosa y de los activos que tiene en su poder. definitivamente no quieren perder.

Aunque la perspectiva de tener que tratar con un actor malicioso que conspira contra ti puede parecer desalentadora, hay 
muchas cosas que puedes hacer para protegerte de todo tipo de esquemas de ingeniería social. En cuanto a la preparación para 

> el lanzamiento de Mainnet, esto debería requerir entrenar tus instintos para tener éxito. detectar y evitar los riesgos de 

Reviewer: ```suggestion
el lanzamiento de la red principal de Cosmos, esto debería requerir entrenar tus instintos para tener éxito, detectar y evitar los riesgos de 
```

> seguridad, seleccionando recursos que sirvan como fuente de verdad para verificar la información, y pasar por algunos pasos 

Reviewer: ```suggestion
seguridad, al seleccionar recursos que sirvan como fuente de verdad para verificar la información, y pasar por algunos pasos 
```

> técnicos para  reducir o eliminar el riesgo de robo de claves o credenciales.

Reviewer: ```suggestion
técnicos para reducir o eliminar el riesgo de robo de claves o credenciales.
```

**Estas son algunas de las reglas de participación que debes tener en cuenta cuando te prepares para el lanzamiento de Cosmos Mainnet:**

> - Descarga software directamente de fuentes oficiales, y asegúrate de que siempre estás usando la última y más segura versión 

Reviewer: ```suggestion
- Descarga _software_ directamente de fuentes oficiales, y asegúrate de que siempre estás usando la última y más segura versión 
```

> de gaiacli cuando estés haciendo algo que incluya tus 12 palabras. Las últimas versiones de Tendermint, el Cosmos-SDK y gaiacli 

Reviewer: ```suggestion
de `gaiacli` cuando estés haciendo algo que incluya tus 12 palabras. Las últimas versiones de Tendermint, la Cosmos-SDK y `gaiacli` 
```

siempre estarán disponibles en nuestros repositorios oficiales de GitHub, y descargándolas desde allí se garantiza que no te 
engañarán para que utilices una versión maliciosamente modificada del software.

> - No comparta sus 12 palabras con nadie. La única persona que debería conocerlos es usted. Esto es especialmente importante 

Reviewer: ```suggestion
- No compartas sus 12 palabras secretas con nadie. La única persona que debería conocerlas eres tú. Esto es especialmente importante 
```

> si alguna vez alguien se le acerca para ofrecerle servicios de custodia para sus Atoms: para evitar perder el control de sus 

Reviewer: ```suggestion
si alguna vez alguien se te acerca para ofrecer servicios de custodia para sus Atoms. P ara evitar perder el control de sus 
```

> tokens, debe almacenarlos fuera de línea para minimizar el riesgo de robo y tener una fuerte estrategia de copia de seguridad 

Reviewer: ```suggestion
criptomonedas, debes almacenarlos fuera de línea (en las denominadas billetera frías) para minimizar el riesgo de robo y tener una fuerte estrategia de copia de seguridad 
```

> en marcha. Y nunca, nunca, nunca los comparta con nadie más.

Reviewer: ```suggestion
en marcha. Y nunca, nunca, nunca los compartas con nadie más.
```

> - Sea escéptico ante adjuntos inesperados o correos electrónicos que le pidan que visite un sitio web sospechoso o 

Reviewer: ```suggestion
- Sé escéptico ante adjuntos inesperados o correos electrónicos que te pidan que visites un sitio web sospechoso o 
```

> desconocido en el contexto de cadenas de bloques o cryptocurrency. Un atacante puede intentar atraerlo a 

Reviewer: ```suggestion
desconocido en el contexto de blockchain o criptomonedas. Un atacante puede intentar atraerte a 
```

> un [sitio comprometido](https://blog.malwarebytes.com/cybercrime/2013/02/tools-of-the-trade-exploit-kits/) diseñado para robar información confidencial de su equipo. Si eres usuario de Gmail, prueba tu 

Reviewer: ```suggestion
un [sitio comprometido](https://blog.malwarebytes.com/cybercrime/2013/02/tools-of-the-trade-exploit-kits/) diseñado para robar información confidencial de tu equipo. Si eres usuario de Gmail, prueba tu 
```

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

> capacidad de recuperación frente a las últimas tácticas de phishing basadas en correo electrónico[aquí](https://phishingquiz.withgoogle.com/).

Reviewer: ```suggestion
capacidad de recuperación frente a las últimas tácticas de _phishing_ basadas en correo electrónico[aquí](https://phishingquiz.withgoogle.com/).
```

> - Haga su debida diligencia antes de comprar Atoms. Los Atoms no serán transferibles en el momento del lanzamiento, por lo 

Reviewer: ```suggestion
- Haz su debida diligencia antes de comprar Atoms. Los Atoms no serán transferibles en el momento del lanzamiento, por lo 
```

> que no pueden ser comprados o vendidos hasta que un hard-fork lo permita. Cuando sean transferibles, asegúrese de que ha 

Reviewer: ```suggestion
que no pueden ser comprados o vendidos hasta que un _hard-fork_ lo permita. Cuando sean transferibles, asegúrate de que haz
```

> investigado al vendedor o intercambiador para confirmar que los Atoms provienen de una fuente confiable.

Reviewer: ```suggestion
investigado al vendedor o casa de cambio para confirmar que los Atoms provienen de una fuente confiable.
```

- Ni el equipo de Tendermint ni la Fundación Interchain venderán Atoms, así que si ves mensajes en los medios sociales o 

> correos electrónicos que anuncian una venta simbólica de nosotros, no son reales y deben ser evitados.  Habilite la 

Reviewer: ```suggestion
correos electrónicos que anuncian una venta simbólica de nosotros, no son reales y deben ser evitados.  Habilita la 
```

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

> autenticación de 2 factores y tenga en cuenta los métodos de recuperación utilizados para recuperar el acceso a sus cuentas 

Reviewer: ```suggestion
autenticación de 2 factores y ten en cuenta los métodos de recuperación utilizados para recuperar el acceso a sus cuentas 
```

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

más importantes. Las cuentas sin protección como el correo electrónico, los medios sociales, tu cuenta de GitHub, el Foro 
Cosmos y cualquier otra cosa en el medio podrían dar a un atacante oportunidades de ganar terreno en tu vida en línea. Si 

> aún no lo ha hecho, empiece a utilizar inmediatamente una aplicación de autenticación o una hardware key dondequiera que 

Reviewer: ```suggestion
aún no lo haz hecho, empieza a utilizar inmediatamente una aplicación de autenticación o una hardware key dondequiera que 
```

Author: **haz** ?

> manejes tus tokens.  Esta es una manera simple, efectiva y comprobada para reducir el riesgo de robo de cuenta.

Reviewer: ```suggestion
manejes tus criptomonedas.  Esta es una manera simple, efectiva y comprobada para reducir el riesgo de robo de cuenta.
```

> - Sea escéptico respecto a los consejos técnicos, especialmente si provienen de personas que no conoce en los foros y en los 

Reviewer: ```suggestion
- Sé escéptico respecto a los consejos técnicos, especialmente si provienen de personas que no conoces en los foros y en los 
```

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

> canales de chat grupal. Familiarícese con los comandos importantes, especialmente aquellos que le ayudarán a llevar a cabo 

Reviewer: ```suggestion
canales de chat grupal. Familiarízate con los comandos importantes, especialmente aquellos que te ayudarán a llevar a cabo 
```

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

> acciones de alto riesgo, y consulte nuestra documentación oficial para asegurarse de que no se le está engañando para que 

Reviewer: ```suggestion
acciones de alto riesgo, y consulta nuestra documentación oficial para asegurarse de que no te están engañando para que 
```

Author: Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.

> haga algo que pueda perjudicarle a usted o a su validador. Y recuerda que el foro del Cosmos, los canales Riot y Telegram no 

Reviewer: ```suggestion
hagas algo que pueda perjudicarte a ti o a tu validador. Y recuerda que el foro del Cosmos, los canales Riot y Telegram no 
```

son fuentes de información oficial o de noticias sobre Cosmos. 

> - Verifique las transacciones antes de pulsar enviar. Sí, esas secuencias de direcciones son largas, pero comparándolas 

Reviewer: ```suggestion
- Verifica las transacciones antes de pulsar enviar. Sí, esas secuencias de direcciones son largas, pero comparándolas 
```

Author: I think when you talk to someone you don't know, (I think) in Spanish it's more correct to talk to them as  _"usted"_ instead of  _"tú"_
But if you think it's more correct like you say I can change it.

visualmente en bloques de 4 carácteres a la vez puede ser la diferencia entre enviarlas al lugar correcto o enviarlas al 
olvido.

> *Si aparece un acuerdo que [suena demasiado bueno para ser cierto][good], o aparece un mensaje pidiendo información que 

Reviewer: ```suggestion
*Si aparece un acuerdo que [suena demasiado bueno para ser cierto][https://www.psychologytoday.com/us/blog/mind-in-the-machine/201712/how-fear-is-being-used-manipulate-cryptocurrency-markets], o aparece un mensaje pidiéndote información que 
```

> nunca, nunca, nunca debe ser compartida con otra persona, siempre puede esforzarse por verificarla antes de comprometerse 

Reviewer: ```suggestion
nunca, nunca, nunca debe ser compartida con otra persona, siempre puedes verificarla antes de comprometerte 
```

> con ella navegando a un sitio web o canal de comunicación oficial de Cosmos por su cuenta. Nadie de Cosmos, el equipo de 

Reviewer: ```suggestion
con ella al visitar un sitio web o canal de comunicación oficial de Cosmos por su cuenta. Nadie de Cosmos, el equipo de 
```

> Tendermint o la Fundación Interchain enviarán alguna vez un email pidiéndole sus 12 palabras para compartirlas con nosotros, 

Reviewer: ```suggestion
Tendermint o la Fundación Interchain enviarán alguna vez un email pidiéndote sus 12 palabras secretas para compartirlas con nosotros, 
```

Author: Same as above, it's more correct to talk to them as _"usted"_  instead of _"tú"_.
But if you think it's more correct like you say I can change it.
